x-pack/plugins/ml/server/routes/datafeeds.ts

```diff
@@ -146,6 +158,9 @@ export function dataFeedRoutes({ router, mlLicense }: RouteInitialization) {
         params: datafeedIdSchema,
         body: datafeedConfigSchema,
       },
+      options: {
+        tags: ['access:ml:canGetJobs'],
```

Comment:
Would `canCreateJob` be better here? Or do we need a new `canCreateDatafeed` one?

Reply:
Historically we've covered creation and deletion of datafeeds with the create and delete job capabilities.
This definitely shouldn't be `canGetJobs`, that is a mistake.

Reply:
i'll add `canCreateDatafeed ` and `canDeleteDatafeed ` for consistency